gstaichi/codegen/cpu/codegen_cpu.cpp

```diff
@@ -53,7 +58,7 @@ class TaskCodeGenCPU : public TaskCodeGenLLVM {
     {
       auto guard = get_function_creation_guard(
           {llvm::PointerType::get(get_runtime_type("RuntimeContext"), 0),
-           llvm::Type::getInt8PtrTy(*llvm_context),
+           llvm::PointerType::get(*llvm_context, 0),
```

Comment:
Do you know if this is compatible with older vesrions of LLVM? Do you know what is the lowest version of LLVM this works for?